release.json

```diff
@@ -8,6 +8,15 @@
 		"JMXFETCH_VERSION": "0.21.0",
 		"JMXFETCH_HASH": "4feb19275604c275b2c1a3b42f788525bb8bcbb3ec14ef62fe26dca1a58c4f99"
 	},
+	"6.7.0-rc.1": {
+		"INTEGRATIONS_CORE_VERSION": "6.6.x",
```

Comment:
This should've been set to a tag in `6.6.0`. It doesn't really matter if `HEAD` hasn't moved, but we should try to keep it in mind to avoid accidentally releasing stuff.

Reply:
Thanks for noticing this. 6.6.x actually got updated for 5.28.1..... 6.6.0 seems to be on the correct tag.